src/lists.rs

```diff
@@ -24,22 +24,71 @@ use utils::{count_newlines, first_line_width, last_line_width, mk_sp, starts_wit
 use visitor::SnippetProvider;
 
 pub struct ListFormatting<'a> {
-    pub tactic: DefinitiveListTactic,
-    pub separator: &'a str,
-    pub trailing_separator: SeparatorTactic,
-    pub separator_place: SeparatorPlace,
-    pub shape: Shape,
+    pub(self) tactic: DefinitiveListTactic,
```

Comment:
Why use `pub(self)` here? Is it not the same as private?

Reply:
Yeah you are right...not sure what I was thinking at the moment 😁 

1a6df18 Fixed.